### PR TITLE
feat: Fail if image manifest does not exist

### PR DESCRIPTION
**Pull request by agatakrajewska**
- [x] Ready for review
- [ ] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
Throw an error for multi arch images, when image manifest specified by `--platform` does not exist.

#### What are the relevant tickets?
https://snyksec.atlassian.net/browse/RUN-1172

### CODE REVIEW

**Review by agranado2k — APPROVED** [review]
LGTM! 🎉

**Review by ivanstanev — APPROVED** [review]

File: test/system/operating-systems/__snapshots__/debian9.spec.ts.snap [hunk 1]
```diff
@@ -3320,18 +3320,18 @@ Object {
           "type": "depGraph",
         },
         Object {
-          "data": "27f9bc730bd1a6a041e45fadd7150e936d585df1521b852d73e441bc1fd1f25f",
+          "data": "c4ccba324c9c4ad861b28a5e5d2e3ad25185dc7752b2f5e2d216566fd4acec7b",
```
Comment (agranado2k) [inline test/system/operating-systems/__snapshots__/debian9.spec.ts.snap:3323]: We should find a solution for this flaky test! :)

**Review by agranado2k — COMMENTED** [review]
One comment! :)

File: test/system/operating-systems/__snapshots__/debian9.spec.ts.snap (hunk 1 above)
Comment (agatakrajewska) [reply test/system/operating-systems/__snapshots__/debian9.spec.ts.snap]: Done :-)

**Review by ivanstanev — DISMISSED** [review]

**Review by ivanstanev — APPROVED** [review]

**Pull request merged**